src/coreclr/src/jit/emitarm64.cpp

```diff
@@ -818,6 +818,22 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             break;
 
         case IF_DV_3C: // DV_3C   .Q.........mmmmm ......nnnnnddddd      Vd Vn Vm   (vector)
+            switch (id->idIns())
+            {
+                case INS_tbl_1reg:
```

Comment:
I would use `INS_tbl` and `INS_tbx` instead of `INS_tbl_1reg` `INS_tbx_1reg` to keep consistency with `ld1` and `st1`.

Reply:
Done!